airflow/utils.py

```diff
@@ -776,3 +778,33 @@ def logger(self):
             self._logger = logging.root.getChild(self.__class__.__module__ + '.' +self.__class__.__name__)
             return self._logger
 
+
+def get_sqla_engine(pool_size=5, pool_recycle=3600):
```

Comment:
It should be parameterized but wasn't / and still isn't. I could parameterized it as part of this PR if you think that's in scope. We probably need to allow different settings for the web server and the scheduler(s).